library/ZendService/Twitter/Response.php

```diff
@@ -43,6 +43,12 @@ class Response
     protected $rawBody;
 
     /**
+     * @var array
+     */
+    protected $rateLimit;
```

Comment:
Hi! :) 

To what end? Yes, it has a specific structure but adding another class to the mix won't really do anything. I would rather simply document the structure. 

=C=